packages/ssr/README.md

```diff
@@ -1 +1,36 @@
 # GCDS Web Components SSR React Wrapper
+
+# GC Design System Components – SSR React Wrapper
+
+GC Design System Components – SSR React Wrapper the GCDS within an SSR Ready React wrapper to ease integration of the components into React applications that utilize the SSR capability.
+
+
+## Documentation
+
+You can find the full documentation for GC Design System Components on [https://design-system.alpha.canada.ca/](https://design-system.alpha.canada.ca/).
+
+## Installation
+
+### Install from npm
+
+Navigate to the root folder of your project and run:
+
+``` js
+npm install @cdssnc/gcds-components @cdssnc/gcds-components-ssr
+// TODO: Review this line above, check if it's correct, or is it just:
+npm install @cdssnc/gcds-components-ssr
+```
```

Comment:
@ethanWallace would you mind double checking if this instruction is correct? thanks!

Reply:
it would be just the bottom one `npm install @cdssnc/gcds-components-ssr`